.github/scripts/check_ep_naming.py

```
@@ -0,0 +1,100 @@
+import re
+import subprocess
+import sys
+
+NAME_RE = re.compile(r"^OSAC-[1-9][0-9]*-[a-z0-9-]+$")
+CHECKED_FILENAMES = frozenset({"prd.md", "design.md"})
+
+BASE_SHA_ENV_VAR = "PRE_COMMIT_PR_BASE_SHA"
+
+
+def path_exists_at_ref(ref: str, path: str) -> bool:
+    result = subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)